### PR TITLE
🎉 Added Conan

### PR DESCRIPTION
**Pull request by AnotherFoxGuy**
Added Conan, the C/C++ Package Manager

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/779)

### CODE REVIEW

**Comment by AnotherFoxGuy** [comment]
Failing due to #259

File: manifests/JFrog/Conan/1.25.2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Id: JFrog.Conan
+Publisher: JFrog Ltd
```
Comment (Reset12138) [inline manifests/JFrog/Conan/1.25.2.0.yaml:2]:
```suggestion
Publisher: JFrog
```

Comment (Reset12138) [reply manifests/JFrog/Conan/1.25.2.0.yaml:2]: that what should be

**Review by Reset816 — CHANGES_REQUESTED** [review]

File: manifests/JFrog/Conan/1.25.2.0.yaml (hunk 1 above)
Comment (Reset12138) [reply manifests/JFrog/Conan/1.25.2.0.yaml:2]: the parent folder should be as same as the Publisher

File: manifests/JFrog/Conan/1.25.2.0.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Id: JFrog.Conan
+Publisher: JFrog
+Name: Conan
+Version: 1.25.2.0
+License: MIT
+InstallerType: inno
+LicenseUrl: https://github.com/conan-io/conan/blob/develop/LICENSE.md
+AppMoniker:
```
Comment (Reset12138) [inline manifests/JFrog/Conan/1.25.2.0.yaml:8]:
```suggestion
```

**Review by Reset816 — CHANGES_REQUESTED** [review]

File: manifests/JFrog/Conan/1.25.2.0.yaml (hunk 2 above)
Comment (Reset12138) [reply manifests/JFrog/Conan/1.25.2.0.yaml:8]: I got into this kind of situation once. It seems that this line should be removed if it's void.

**Comment by AnotherFoxGuy** [comment]
302 are still broken

**Comment by SoftCreatR** [comment]
@AnotherFoxGuy Meanwhile, you can use this url instead:

`https://www.winget.it/dl/af829b7ca8d81e11cab152c488daae88a45bbd3e6c5a27258a4ec0332ea433eb/https%3A%2F%2Fdl.bintray.com%2Fconan%2Finstallers%2Fconan-win-64_1_25_2.exe`

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**